projects/commudle-admin/src/app/feature-modules/community-builds/components/create-community-build/create-community-build.component.ts

```diff
@@ -16,11 +16,12 @@ import { LibToastLogService } from 'projects/shared-services/lib-toastlog.servic
 export class CreateCommunityBuildComponent implements OnInit {
 
   cBuild: ICommunityBuild;
-  tags = '';
+  tags: string[] = [];
   linkFieldLabel = 'Any Link?';
   EBuildType = EBuildType;
   EPublishStatus = EPublishStatus;
   redirectTo;
+  emails: {value: string} [] = [];
```

Comment:
We discussed that we need to use reactive forms, to dynamically add teammate email inboxes so that it gets submitted within the form itself....
Please fix this.

Reply:
okkk I will fix it  :)